fenix/app/src/main/res/layout/studies_section_item.xml

```diff
@@ -22,12 +22,12 @@
         android:layout_marginStart="@dimen/top_bar_alignment_margin_start"
         android:id="@+id/title"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/section_header_height"
+        android:layout_height="wrap_content"
```

Comment:
Nice catch!
We see now that using a hardcoded size for layout of a text should be avoided.